.travis.yml

```diff
@@ -2,4 +2,6 @@ sudo: false
 language: ruby
 rvm:
   - 2.3.3
-before_install: gem install bundler -v 1.17.3
+  - 2.4.6
+  - 2.5.1
+before_install: gem update --system --no-document && gem install bundler -v '~> 2.0'
```

Comment:
Why are you changing the bundler? The Gemfile.lock is still referencing `1.17.3`

Reply:
Like I mentioned in the commit, bundlers > 2.0 automatically shift to the version listed in the lockfile.  You don't need to install the specific version of bundler anymore.